File: Assignment_No-35/Question_02.java
/*2)write a program which accept string form user and count number of small Character

input:"Marvellous"

output:9
*/

///////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Demo
{
    public int CountSmall(String str)
	{

		int iCount=0;
    
		int i=0;
    
		for(i=0;i<str.length();i++)
		{
			if(Character.isLowerCase(str.charAt(i)))
			{
				iCount++;
			}
      
		}
		return iCount;

	}
   
}

class ass177
{
    public static void main(String A[])
    {
        
         int iRet=0;
         
		     Scanner sobj=new Scanner(System.in);


		    System.out.println("Enter the name of string:");
		    String str=sobj.nextLine();
     

        Demo dobj=new Demo();

        iRet=dobj.CountSmall(str);

        System.out.println("Small character is:"+iRet);
 }

}

File: Assignment_No-27/Question_02.c
/*2)Accept a program which accept string from user and count number of small characters.
input:Marvellous 

output:9

*/

///////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

int CountSmall(char *str)
{

	int iCount=0;
	while(*str!='\0')
	{

		if(*str>='a' && *str<='z')
		{

			iCount++;
		}
		str++;
	}
    
    return iCount;
}

int main()
{

	char Arr[20];
	int iRet=0;
  
	printf("Enter a string:\n");
	scanf("%[^'\n']s",Arr);

	iRet=CountSmall(Arr);

	printf("%d",iRet);
  
	return 0;

}
